api/src/main/java/org/pmiops/workbench/firecloud/FireCloudConfig.java

```diff
@@ -102,4 +104,20 @@ public GroupsApi groupApi(@Qualifier(END_USER_API_CLIENT) ApiClient apiClient) {
     return api;
   }
 
+  @Bean
+  @RequestScope(proxyMode = ScopedProxyMode.DEFAULT)
+  public StatusApi statusApi(WorkbenchConfig workbenchConfig) {
+    StatusApi statusApi = new StatusApi();
+    ApiClient apiClient = new ApiClient();
+    apiClient.setBasePath(workbenchConfig.firecloud.baseUrl);
+    apiClient.setDebugging(workbenchConfig.firecloud.debugEndpoints);
+    addFireCloudDefaultHeader(apiClient);
+    statusApi.setApiClient(apiClient);
+    return statusApi;
+  }
+
+  private void addFireCloudDefaultHeader(ApiClient apiClient) {
+    apiClient.addDefaultHeader("X-App-ID", "AoU-RW");
```

Comment:
Do we need to have a different app ID for different environments?

Could define constant or config setting for this....